### ISSUE TITLE
[a11] Links do not have a discernible name

### ISSUE THREAD

**Issue by rjbeers** [codeconnector/codeconnective.github.io#25]
[This lighthouse report](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fcodeconnector.io%2F#accessibility) shows that icons used as links fail accessibility requirements by not having discernible names.

> Link text (and alternate text for images, when used as links) that is discernible, unique, and focusable improves the navigation experience for screen reader users.

See [Link Name documentation](https://web.dev/link-name/) for more details.
